app/helpers/katello/subscription_mailer_helper.rb

```diff
@@ -0,0 +1,17 @@
+module Katello
+  module SubscriptionMailerHelper
+    include Rails.application.routes.url_helpers
+
+    def report_url
```

Comment:
This method is doing two things - running the report and building the url.  I think it should be broken into two methods.  `report_url` can call the `start_report_task` method.